repository/src/main/resources/alfresco/repository.properties

```diff
@@ -1209,6 +1209,10 @@ repo.event2.filter.childAssocTypes=rn:rendition
 repo.event2.filter.users=System, null
 # Topic name
 repo.event2.topic.endpoint=amqp:topic:alfresco.repo.event2
+# Thread pool for async delivery
+repo.event2.threadPool.priority=1
+repo.event2.threadPool.coreSize=8
+repo.event2.threadPool.maximumSize=10
 

```

Comment:
Check if you can get rid of this ^M characters

Reply:
The file is filled with that line termination character, I'd leave it as is otherwise I will have to remove it entirely.